pandas/errors/__init__.py

```diff
@@ -0,0 +1,60 @@
+# flake8: noqa
+
+""" expose public exceptions & warnings """
+
+from pandas._libs.tslib import OutOfBoundsDatetime
+
+
+class PandasError(Exception):
+    pass
```

Comment:
hmm, actually I think I'll leave ``PandasError`` not exposed then. We can always add it later (if needed).